src/libLookup/Lookup.h

```diff
@@ -45,25 +45,41 @@ class Lookup : public Executable, public Broadcastable
 {
     Mediator& m_mediator;
 
-#ifndef IS_LOOKUP_NODE
     // Info about lookup node
     std::vector<Peer> m_lookupNodes;
+    std::vector<Peer> m_lookupNodesOffline;
     std::vector<Peer> m_seedNodes;
-
+#ifndef IS_LOOKUP_NODE
+    bool m_dsInfoWaitingNotifying = false;
     bool m_fetchedDSInfo = false;
     std::mutex m_mutexDSInfoUpdation;
-    std::condition_variable m_dsInfoUpdateCondition;
+    std::condition_variable cv_dsInfoUpdate;
```

Comment:
Please follow coding rule: use m_xxx as class member variable

Reply:
Junhao asked me to use cv_xxx

Reply:
why?

Reply:
I think right now, all the conditional variable start with cv_. We haven't fully define the coding convention. So for consistency with the code base we have, I have asked @kaikawaliu to change it to cv_. 

In the future, if we have a comprehensive coding convention, we can mass refactor all the conditional variable.

Reply:
ok